src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -16,7 +17,7 @@ import scala.util.Try
 
 object Backend {
   lazy val LocalBackend = new LocalBackend
-  lazy val JesBackend = new JesBackend { JesConf } //forces configuration resolution to fail now if something is missing
+  lazy val JesBackend = new JesBackend with JesAuthenticationSingleton { JesConf } //forces configuration resolution to fail now if something is missing
```

Comment:
Trying to understand this `with JesAuthenticationSingleton` pattern.  Is this so you can use a mocked implementation of some of this functionality in the test?  If so, couldn't you fold this behavior into `JesBackend` and still accomplish the same thing by stacking on the mock trait?